### PR TITLE
Disable indexing based on playbook inputs

### PR DESCRIPTION
**Pull request by naveena-maplelabs**
1) After upgrading to 6.5.1 version, customer faced issues while
creating jobs.
2) Indexing is diabled by default.
3) Added new field in the tasks to support diableIndexing

### CODE REVIEW

File: tasks/job.yml [hunk 1]
```diff
@@ -21,4 +21,5 @@
     time_zone: "{{ cohesity_protection.time_zone | default('America/Los_Angeles') }}"
     start_time: "{{ cohesity_protection.start_time | default('') }}"
     delete_sources:  "{{ cohesity_protection.delete_sources | default(False) }}"
+    disable_indexing: "{{ cohesity_protection.disable_indexing | default(True) }}"
```
Comment (pyashish) [inline tasks/job.yml]: Check if this is applicable to vmware /physical/nas/oracle .. the supported configs..else we may have to : 
1. Enhance documentation 
2. Validations in code to catch any other env. is given for indexing

**Review by pyashish — COMMENTED** [review]
The summary comment isn't correct. 

Indexing is enabled by default ?

File: tasks/job.yml (hunk 1 above)
Comment (naveena-maplelabs) [reply tasks/job.yml]: Updated documentation

File: library/cohesity_job.py [hunk 2]
```diff
@@ -496,6 +496,39 @@ def get_view_storage_domain_id(module, self):
     except Exception as error:
         raise__cohesity_exception__handler(error, module)
 
+def update_indexing(module, payload):
+    """
+    """
+    payload['indexingPolicy'] = {
+        "disableIndexing": module.params.get('disable_indexing'),
+        "allowPrefixes": ["/"],
```
Comment (pyashish) [inline library/cohesity_job.py:504]: These defaults are mentioned in the documentation right ?

Comment (pyashish) [reply library/cohesity_job.py:504]: Will update functionality cause override or append to existing one. Either ways make the documentation clear.

Comment (naveena-maplelabs) [reply library/cohesity_job.py:504]: Will override existing values. I have updated the same in documentation as well.

**Pull request merged**